### PR TITLE
reduced the len of s3 bucket name for docker images replication

### LINKED ISSUE #89: [BUG] Replication Module Bucket Name is too long
**Describe the bug**
Deployment of the replication module fails in eu-central-1 due to bucket name length exceeded error.

https://github.com/awslabs/idf-modules/blob/b2530110267ba9315a34a0ccfafc5e0ff15b0374/modules/replication/dockerimage-replication/deployspec.yaml#L10

**To Reproduce**
Deploy replication module to eu-central-1

**Expected behavior**
Module deploys without a problem

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by srinivasreddych**
*Issue #, if available:*
Fixes #89 
*Description of changes:*
reduced the len of s3 bucket name for docker images replication

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: modules/compute/eks/modulestack.yaml [hunk 1]
```diff
@@ -35,12 +35,12 @@ Resources:
             Action:
               - "s3:ListBucket"
             Resource:
-              - "arn:aws:s3:::*-dockerimages-replications-*"
+              - "arn:aws:s3:::*-dkr-img-rep-*"
           - Effect: Allow
             Action:
               - "s3:GetObject"
             Resource:
```
Comment (dgraeber) [inline modules/compute/eks/modulestack.yaml:42]: Can't do this....you need to support backward compatibility, so both versions of the bucket name need to be in the policy, else users will be hosed..

File: modules/replication/dockerimage-replication/modulestack.yaml [hunk 2]
```diff
@@ -45,13 +45,13 @@ Resources:
...
               - "s3:PutObject"
             Effect: Allow
-            Resource: "arn:aws:s3:::*-dockerimages-replications-*"
+            Resource: "arn:aws:s3:::*-dkr-img-rep-*"
           - Action:
               - "s3:CreateBucket"
               - "s3:ListBucket"
               - "s3:PutObject"
             Effect: Allow
-            Resource: "arn:aws:s3:::*-dockerimages-replications-*/*"
```
Comment (dgraeber) [inline modules/replication/dockerimage-replication/modulestack.yaml:54]: Can't do this....you need to support backward compatibility, so both versions of the bucket name need to be in the policy, else users will be hosed..

**Review by dgraeber — CHANGES_REQUESTED** [review]
The policy needs to now support both bucket name formats for backward compatibility...

File: modules/compute/eks/modulestack.yaml (hunk 1 above)
Comment (srinivasreddych) [reply modules/compute/eks/modulestack.yaml:42]: Good catch!

**Review by dgraeber — APPROVED** [review]

**Pull request merged**